.pre-commit-config.yaml

```
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-case-conflict
+    -   id: check-merge-conflict
+    -   id: requirements-txt-fixer
+    -   id: trailing-whitespace
+        exclude: ^backend/static/
+# Note that this does not mix with git setup check-in LF check-out CRLF,
+# see https://github.com/pre-commit/pre-commit-hooks#mixed-line-ending
+    -   id: mixed-line-ending
+        args: [--fix=lf]
+        exclude: ^backend/static/
+# use black or autopep8 to automatically format files to pep8
+-   repo: https://github.com/psf/black
+    rev: stable
+    hooks:
+    -   id: black
+        exclude: ^backend/apps/[a-z]*/migrations/[0-9A-z]*.py
```

Add additional `language_version` and set to 3.6 (is the minimum required version for Django 3.0.5)

Further material:
https://black.readthedocs.io/en/stable/version_control_integration.html
or `target_version` in the cli version https://pypi.org/project/black/

```suggestion
        language_version: python3.6
        exclude: ^backend/apps/[a-z]*/migrations/[0-9A-z]*.py
```

---

Actually, this does more harm than good and causes bug #482. Is there an important reason to add this? Black works fine without, and this would not be the place to add this. In order to be a version check, it would need to be its own hook. @kevihiiin